### PR TITLE
Samsp/configfiles

### PR DESCRIPTION
**Pull request by samsp-msft**

### CODE REVIEW

File: docs/docfx/articles/configfiles.md [hunk 1]
```diff
@@ -83,3 +83,122 @@ The proxy will apply the given matching criteria and policies, and then pass off
 The clusters section is an unordered collection of named clusters. A cluster primarily contains a collection of named destinations and their addresses, any of which is considered capable of handling requests for a given route. The proxy will process the request according to the route and cluster configuration in order to select a destination.
 
 For additional fields see [Cluster](xref:Microsoft.ReverseProxy.Abstractions.Cluster).
+
+## All config properties
+
+{
```
Comment (Tratcher) [inline docs/docfx/articles/configfiles.md:89]: This section needs to be in a ``` block.

I can see why you put this here, but how to we make sure it is kept up to date?

**Comment by Tratcher** [comment]
Contributes to https://github.com/microsoft/reverse-proxy/issues/751

**Comment by Tratcher** [comment]
This needs to rebase now that I've changed the project names.

**Comment by samsp-msft** [comment]
CP'd main changes

**Pull request closed without merging**